source/_integrations/airgradient.markdown

```diff
@@ -45,10 +45,20 @@ The integration will fetch data from each device. The following sensors are supp
 - PM10 density
 - Raw nitrogen
 - Raw total volatile organic compounds
+- Raw PM2.5
 - Signal strength
 - Temperature
 - Total volatile organic compounds index
 
+A number of configuration entities are available as sensors to automate with if you set up the device via AirGradient Cloud instead of setting it up locally.
```

Comment:
```suggestion
A number of configuration entities are available as sensors to automate with if you control the device via the AirGradient cloud instead of set it to control locally.
```